admin/fleetcommander/fcdbus.py

```diff
@@ -198,43 +199,50 @@ def run(self, sessionbus=False):
         self._loop = GObject.MainLoop()
 
         # Prepare changes listener
-        self.changeslistener_port = self.get_free_port()
-        self.changeslistener = Soup.Server()
-        self.changeslistener.listen_all(
-            self.changeslistener_port, Soup.ServerListenOptions.IPV4_ONLY)
-        self.changeslistener.add_handler(
+        self.webservice_port = self.DEFAULT_WEBSERVICE_PORT
+        self.webservice = Soup.Server()
+        try:
+            self.webservice.listen_all(
```

Comment:
we should make the listen host/port configurable through /etc/xdg/fleet-commander-admin.conf